src/cljs/pb/views/proposals.cljs

```diff
@@ -1,21 +1,42 @@
 (ns pb.views.proposals
   (:require [re-frame.core :as rf]
-            [pb.components.proposal :refer [proposal-component]]))
+            [pb.components.proposal :refer [proposal-component]]
+            [clojure.string :as string]))
+
+(defn query [ids]
+  (let [queries (for [id ids]
+                  (str "p" id ": proposalAS(q: \"sys.id="
+                       id
+                       "\") {
+                         title
+                         objective
+                         impact
+                         budget
+                         timeline
+                         images { url }
+                       }"))
+        combined-query (str "{" (string/join queries) "}")]
+      combined-query))
```

Comment:
since we only use `combined-query`  once we can probably just do:
```
(defn query [ids]
  (let [queries (for [id ids]
                  (str "p" id ": proposalAS(q: \"sys.id="
                       id
                       "\") {
                         title
                         objective
                         impact
                         budget
                         timeline
                         images { url }
                       }"))]
      (str "{" (string/join queries) "}")))
```